library/CMakeLists.txt

```diff
@@ -714,6 +714,10 @@ if(TINYSPLINE_CXX_AVAILABLE AND TARGET_SUPPORTS_SHARED_LIBS)
 						COMPILE_FLAGS "${TINYSPLINE_BINDING_CXX_FLAGS} ${PHP_INCLUDE_DIRS}"
 					)
 				endif()
+			else()
+				if(APPLE)
+					message(STATUS "Note: the PHP version that comes with macOS does not have required headers.")
```

Comment:
Since we are looking for Zend, this message could be somewhat more specific. Maybe:

"The PHP interface requires Zend which has not been found in the default include path. Note: Some operating systems, for instance, OS X, bundle PHP without Zend."

Reply:
I like that, yeah -- fixed!